web/themes/custom/usagov/scripts/telephoneForMobileNode.js

```diff
@@ -0,0 +1,15 @@
+jQuery(document).ready(function ($) {
+  if (window.innerWidth <= 950) {
+    $(".field--type-telephone").replaceWith(function () {
+      return $(
+        "<a href='tel:" + $(this).html() + "'>" + $(this).html() + "</a>"
+      );
+    });
+  }
+
+  if (window.innerWidth >= 950) {
```

Comment:
From what I'm seeing, we don't have anything triggering this script when the viewport changes, so I don't believe there's any need to "reset" the links.

Reply:
... which I now notice was in the example from mothership, but I guess I wasn't in review mode when I looked at that!

Reply:
I'll update this as well